config/application.rb

```diff
@@ -97,7 +97,7 @@ class Application < Rails::Application
 
     config.domain_event_handlers = {
       'offender-management.noop' => 'DomainEvents::Handlers::NoopHandler',
-      'prisoner-offender-search.prisoner.updated' => 'DomainEvents::Handlers::PrisonerUpdatedHandler',
+      # 'prisoner-offender-search.prisoner.updated' => 'DomainEvents::Handlers::PrisonerUpdatedHandler',
```

Comment:
I assume this is an intentional commenting-out?

Reply:
Yep, I'll refer you to the commit where the change was made for the explanation